test/__mocks__/text.mock.ts

```diff
@@ -9,7 +9,7 @@ export const BENEFICIAL_OWNER_GOV_PAGE_HEADING = "Tell us about the government o
 export const BENEFICIAL_OWNER_OTHER_PAGE_HEADING = "Tell us about the other legal entity";
 export const BENEFICIAL_OWNER_STATEMENTS_PAGE_REDIRECT = "Found. Redirecting to /register-an-overseas-entity/beneficial-owner-statements";
 export const BENEFICIAL_OWNER_TYPE_PAGE_REDIRECT = "Found. Redirecting to /register-an-overseas-entity/beneficial-owner-type";
-export const BENEFICIAL_OWNER_STATEMENTS_PAGE_HEADING = "Have you identified any beneficial owners?";
```

Comment:
Looks fine, just wondering why the mock did not match the title before as they it does now.

Reply:
True it's an odd one, if I change the unit tests to something else however it does fail

Reply:
Its referring to the hidden field [here](https://github.com/companieshouse/overseas-entities-web/blob/f7f6f8517b50738b90679e0a58b83ef3ea9f8143/views/beneficial-owner-statements.html#L38) that I think should be modified as well, text should be `Has the overseas entity identified any registrable beneficial owners?`, same as the title.

Reply:
Better if you use the title variable:
```
...
  legend: {
    text: title,
    isPageHeading: false,
    classes: "govuk-visually-hidden"
  }
...
```